haggis.js

```diff
@@ -554,3 +563,34 @@ function extractTableId(doc) {
 function sortNumeric(a, b) {
   return a - b;
 }
+
+function addPointsPerRound(game) {
+    const ptsPerRound = getTotalPointsAfterEachRound(game.rounds);
+    
+    game.rounds.forEach((_, i) => {
+        if (i >= game.rounds.length - 1) { 
+            return;
+        }
```

Comment:
You don't have to change this, but, personally, I think an og-style for loop is more clear here than the `forEach`